js/src/frameworks/langgraph.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { describe, it, expect, beforeAll } from "@jest/globals";
+import { z } from "zod";
+import { getTestConfig } from "../../config/getTestConfig";
+import { LanggraphToolSet } from "./langgraph";
+
+
+// Langgraph extens langchain class, all the properties are same
+describe("Apps class tests", () => {
+    it("getools", async () => {
+        let langgraphToolSet = new LanggraphToolSet({
+            apiKey: getTestConfig().COMPOSIO_API_KEY,
+            baseUrl: getTestConfig().BACKEND_HERMES_URL,
+        });
+        const tools = await langgraphToolSet.getTools({
+            apps: ["github"],
+        });
+        expect(tools).toBeInstanceOf(Array);
+    });
+})
```

Comment:
The test coverage is minimal. Consider adding tests for:
1. Error cases (invalid API key, invalid base URL)
2. Different configuration options (with/without entityId, workspaceConfig)
3. Runtime parameter validation
4. Framework name constant verification